kpet/data.py

```diff
@@ -451,6 +474,56 @@ def convert_tree_arches(self):
 
             self.trees[name]["arches"] = list(tree_arches)
 
+    def convert_sets(self):
+        """ Convert each suite / cases sets from a list of regexes
```

Comment:
Could you start the description from the new line, like the rest of docstrings does?